quickwit/quickwit-indexing/src/source/mod.rs

```diff
@@ -115,9 +116,11 @@ impl SourceExecutionContext {
         queues_dir_path: PathBuf,
         source_config: SourceConfig,
     ) -> Arc<SourceExecutionContext> {
+        use ulid::Ulid;
+
         Arc::new(Self {
             metastore,
-            index_id: index_id.to_string(),
+            index_uid: IndexUid::from_parts(index_id, Ulid::nil()),
```

Comment:
I think it might be better to pass `index_uid` here instead of generating a fake one. I tried the fake generation approach in #3212 but very quickly regretted it except for very small unit tests.

Reply:
Ok!